seed/views/v3/labels.py

```diff
@@ -0,0 +1,114 @@
+# !/usr/bin/env python
+# encoding: utf-8
+"""
+:copyright (c) 2014 - 2020, The Regents of the University of California, through Lawrence Berkeley National Laboratory (subject to receipt of any required approvals from the U.S. Department of Energy) and contributors. All rights reserved.  # NOQA
+:author 'Piper Merriam <pmerriam@quickleft.com>'
+"""
+from docutils.nodes import status
+from drf_yasg.utils import swagger_auto_schema
+from rest_framework import response, status
+from rest_framework.decorators import action
+from rest_framework.parsers import JSONParser, FormParser
+from rest_framework.renderers import JSONRenderer
+
+from seed.decorators import DecoratorMixin
+from seed.filters import (
+    LabelFilterBackend,
+)
+from seed.models import (
+    StatusLabel as Label,
+)
+from seed.serializers.labels import (
+    LabelSerializer,
+)
+from seed.utils.api import drf_api_endpoint
+from django.utils.decorators import method_decorator
+from seed.utils.api_schema import AutoSchemaHelper
+from seed.utils.labels import filter_labels_for_inv_type
+from seed.utils.viewsets import SEEDOrgNoPatchOrOrgCreateModelViewSet
+
+
+@method_decorator(
+    name='retrieve',
+    decorator=swagger_auto_schema(
+        manual_parameters=[AutoSchemaHelper.query_org_id_field(required=False)]
+    ),
+)
+@method_decorator(
+    name='list',
+    decorator=swagger_auto_schema(
+        manual_parameters=[AutoSchemaHelper.query_org_id_field(required=False)]
+    ),
+)
+@method_decorator(
+    name='create',
+    decorator=swagger_auto_schema(
+        manual_parameters=[AutoSchemaHelper.query_org_id_field(required=False, description="put in org id description")],
```

Comment:
Reminder for this change request on this and other `query_org_id_field`s. The description should mention that  the param is optional and will default to the user's currently selected organization.